plugins/sfDcPlugin/modules/sfDcPlugin/actions/dcComponent.class.php

```diff
@@ -24,5 +24,9 @@ public function execute($request)
     $this->iso639convertor = new fbISO639_Map;
 
     $this->dc = new sfDcPlugin($this->resource);
+
+    // Get creators, inheriting if needed
+    $options = array('eventTypeId' => QubitTerm::CREATION_ID, 'inherit' => true);
```

Comment:
I think it would be more consistent to put this logic in https://github.com/artefactual/atom/blob/qa/2.6.x/plugins/sfDcPlugin/lib/sfDcPlugin.class.php in the `__get()` magic accessor method.

Reply:
Actually, one more thing - I don't think assigning the `$options` array is necessary or enhances readability here.  I would move the `array()` constructor into the `getActors()` call.